packages/server/lib/screenshots.ts

```diff
@@ -468,8 +487,10 @@ module.exports = {
 
       return getBuffer(details)
       .then((buffer) => {
+        // @ts-ignore
         return fs.outputFileAsync(pathToScreenshot, buffer)
       }).then(() => {
+        // @ts-ignore
```

Comment:
while `outputFile` is an additional method that `fs-extra` adds, this should be able to be resolved by `import { stat } from 'fs/promises'`

Reply:
[`7d6e3a0` (#30758)](https://github.com/cypress-io/cypress/pull/30758/commits/7d6e3a0b960a487f596937ecb85371a3e39534eb)